cmd/createcluster.go

```diff
@@ -91,6 +94,7 @@ func newCreateClusterCmd(runFunc func(context.Context, io.Writer, clusterConfig)
 func bindClusterFlags(flags *pflag.FlagSet, config *clusterConfig) {
 	flags.StringVar(&config.Name, "name", "", "The cluster name")
 	flags.StringVar(&config.ClusterDir, "cluster-dir", ".charon/cluster", "The target folder to create the cluster in.")
+	flags.StringVar(&config.DefFile, "definition-file", "", "The path to the file or an HTTP URL to fetch cluster definition.")
```

Comment:
`--definition-file: Optional path to a cluster definition file or an HTTP URL. This overrides all other configuration flags.`